.github/workflows/spirv-tests.yml

```diff
@@ -9,6 +9,7 @@ on:
     paths:
       - 'llvm/lib/Target/SPIRV/**'
       - 'llvm/test/CodeGen/SPIRV/**'
+      - 'mlir/test/Target/SPIRV/**'
```

Comment:
I think we also need the other spirv paths, including dialect definition and conversions

Reply:
Why? They do not generate SPIR-V assembly, so they do not require spirv-tools support. Or am I missing something?

Reply:
Also it's not just Target tests, also the serializer/deserializer implementation

Reply:
Updated